src/java.security.jgss/share/classes/sun/security/krb5/internal/Krb5.java

```diff
@@ -270,13 +273,13 @@ public class Krb5 {
     public static final int KRB_AP_ERR_METHOD            = 48;   //Alternative authentication method required
     public static final int KRB_AP_ERR_BADSEQ            = 49;   //Incorrect sequence number in message
     public static final int KRB_AP_ERR_INAPP_CKSUM       = 50;   //Inappropriate type of checksum in message
+    public static final int KRB_AP_PATH_NOT_ACCEPTED     = 51;   //Policy rejects transited path
     public static final int KRB_ERR_RESPONSE_TOO_BIG     = 52;   //Response too big for UDP, retry with TCP
     public static final int KRB_ERR_GENERIC              = 60;   //Generic error (description in e-text)
     public static final int KRB_ERR_FIELD_TOOLONG        = 61;   //Field is too long for this implementation
     public static final int KRB_ERR_WRONG_REALM          = 68;   //Wrong realm
-    public static final int KRB_CRYPTO_NOT_SUPPORT      = 100;    //Client does not support this crypto type
-    public static final int KRB_AP_ERR_NOREALM          = 62;
-    public static final int KRB_AP_ERR_GEN_CRED         = 63;
+
+    public static final int KRB_CRYPTO_NOT_SUPPORT      = 100;   //Client does not support this crypto type
```

Comment:
Just wondering, and not a specific review comment, but do you know if this error code is necessary? It isn't defined in RFC 4120 and not used in the JDK AFAICT.